org.gnome.Sudoku.json

```diff
@@ -0,0 +1,44 @@
+{
+    "app-id": "org.gnome.Sudoku",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.32",
+    "sdk": "org.gnome.Sdk",
+    "command": "gnome-sudoku",
+    "finish-args": [
+        /* X11 + XShm access */
+        "--share=ipc", "--socket=x11",
+        /* Wayland access */
+        "--socket=wayland",
+        /* dconf */
+        "--filesystem=xdg-run/dconf", "--filesystem=~/.config/dconf:ro",
+        "--talk-name=ca.desrt.dconf", "--env=DCONF_USER_CONFIG_DIR=.config/dconf"
+    ],
+    "modules": [{
+            "name": "libgee",
```

Comment:
This should be in the 3.32 runtime. Try removing it!

Reply:
Wasn't aware of that, will remove it.

Reply:
Still the build fails due to missing release tags.